### ISSUE TITLE
UI component: UBQ bonding curve to burn UBQ and mint NFTs

### ISSUE THREAD

**Issue by sergfeldman** [pavlovcik/uad-ui#429]
To do 
Update UI component according to the discussion 
- https://github.com/ubiquity/ubiquity-dollar/discussions/428

The issue is related to 
- https://github.com/ubiquity/ubiquity-dollar/issues/430
- https://github.com/ubiquity/ubiquity-dollar/issues/434

**Comment by milos04** [comment]
Hello @sergfeldman I'd like to work on this issue if there is no blockers, and hope the issue description will be described again here. Not easy to get the main points

**Comment by sergfeldman** [comment]
> Hello @sergfeldman I'd like to work on this issue if there is no blockers, and hope the issue description will be described again here. Not easy to get the main points

@milos04 Hello, can you please read the description and post your your questions if they are?
https://github.com/ubiquity/ubiquity-dollar/discussions/428

**Comment by 0x4007** [comment]
@sergfeldman I think you should rewrite the description because this is really hard to follow. Please compile all the relevant information for this $100 bounty (that is estimated to take les than one day to complete) and making the objective concise. You can refer to my bounties I wrote for examples.

**Comment by FibrinLab** [comment]
@pavlovcik Would love to work on this. Can i get all relevant information before i begin?

**Comment by 0x4007** [comment]
I think the relevant information is in https://github.com/ubiquity/ubiquity-dollar/discussions/428

**Comment by isreallee82** [comment]
is this still available?
